src/main/kotlin/blackjack/model/Participants.kt

```diff
@@ -10,15 +10,14 @@ class Participants(
 
     fun getAllParticipants(): List<Participant> = listOf(dealer) + players
 
-    fun getPlayerWinningState(): Map<Player, WinningResult> {
-        return players.associateWith { player -> player.getWinningResult(dealer) }
+    fun getPlayerProfits(): Map<Player, Double> {
+        return players.associateWith { player ->
+            val winningResult = player.getWinningResult(dealer)
+            player.calculateProfit(winningResult)
+        }
     }
 
-    fun getDealerWinningState(): Map<WinningResult, Int> {
-        return players.map { player ->
-            dealer.getWinningResult(player)
-        }.groupingBy { it }.eachCount()
-    }
+    fun getDealerProfits(playersResult: Map<Player, Double>): Double = playersResult.entries.sumOf { it.value } * (-1)
```

Comment:
```suggestion
    fun getDealerProfits(playersResult: Map<Player, Double>): Double = playersResult.entries.sumOf { it.value } * (-1)
```
result 보다는 기존에 적어주신 profit 이라고 네이밍이 지어져야 명확할 것 같습니다!

Reply:
players 의 수익을 외부에서 전달 받는 것이 아니라, 내부에서 바로 참조할 수 있지 않을까요?

Reply:
플레이어와 딜러의 수익을 합쳐서 반환하도록 수정했습니다~